### ISSUE TITLE
Feature request: Merge secondary repos into xmtp-web

### ISSUE THREAD

**Issue by neekolas** [xmtp/xmtp-web#97]
### Is your feature request related to a problem?

We have a lot of "support repos" for the web. 
- https://github.com/xmtp/xmtp-js-content-types
- https://github.com/fabriguespe/xmtp-quickstart-reactjs
- https://github.com/xmtp/xmtp-react-playground
- https://github.com/xmtp/react-editable-interactive-tutorial
- https://github.com/fabriguespe/xmtp-subscribe-push-demo

These repos all work in subtly different ways, with different `tsconfig` settings, lint rules, and build processes.

### Describe the solution to the problem

We could benefit from consolidated tsconfig, build, lint, and test infrastructure by consolidating them into this monorepo. We have already done the work of making this repo work efficiently as a monorepo. 

### Describe the uses cases for the feature

Make it easier for developers to experiment with XMTP

**Comment by fabriguespe** [comment]
It's an interesting idea! However, I have some thoughts about why keeping the repositories separate might be more beneficial, especially for our DevRel efforts. These standalone repos are helpful when assisting beginner developers who are getting started with XMTP. They often seek quick, specific guidance, and these granular quickstarts seem to help them get unblocked swiftly.

Combining everything into a monorepo could potentially introduce more complexity, which might make it more difficult for beginners to understand individual features. Also, each of these repos is quite simple in terms of build and lint requirements. They primarily serve as functional examples for developers to reference and build upon. 

Since many of them use similar components, maintaining them separately hasn't been a big challenge. Just wanted to share this perspective before we make any decisions!

**Comment by rygine** [comment]
i'm going to close this issue as `Won't Do` for now. we can explore this option again in the future if it makes sense.
